.golangci.yml

```diff
@@ -37,8 +37,9 @@ linters-settings:
     include-go-root: true # check against stdlib
     packages-with-error-message:
     - io/ioutil: 'use "io" or "os" packages instead'
-    - github.com/siddontang/go/log: 'use "github.com/sirupsen/logrus" instead'
+    - github.com/golang/protobuf: 'use "github.com/golang/protobuf"'
```

Comment:
Am I missing something or are these the same?

Reply:
Done!